frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -607,8 +612,8 @@ export const NamespacedClusterServiceVersionList: React.SFC<ClusterServiceVersio
           subscriptions={subscriptions.data}
         />
       )}
-      EmptyMsg={NoOperatorsMatchFilterMsg}
-      NoDataEmptyMsg={NoDataEmptyMsg}
+      EmptyMsg={() => <MsgBox title={t('olm~No Operators found')} />}
```

Comment:
It would be better to make this a separate component outside of this one instead of an inline component so it doesn't remount on every render.